### ISSUE TITLE
Stopping a receiving messages when an error occurs

### ISSUE THREAD

**Issue by SijyKijy** [TelegramBots/Telegram.Bot.Extensions.Polling#12]
When an error occurs in `HandleUpdateAsync` and `HandleErrorAsync` methods, bot stop receiving messages.
Is that how it's supposed to work?

https://github.com/TelegramBots/Telegram.Bot.Extensions.Polling/blob/52c1edf4ff13d97c0b6d5ed8a62631cceadd9210/src/Telegram.Bot.Extensions.Polling/DefaultUpdateReceiver.cs#L99-L115

**Comment by tuscen** [comment]
Yes, that's the expected behaviour, it's the same as with previous events in the client.

If there's an exception in user code we can't do anything other than stop execution.

**Comment by SijyKijy** [comment]
How can I restart receiver If an error occurs?
